### PR TITLE
zh-trans: add content/case-studies/zalando/index.html

### PR DESCRIPTION
**Pull request by ten2ton**
zh-trans: add content/case-studies/zalando/index.html

### CODE REVIEW

**Comment by sftim** [comment]
The changes are different from the description as there are also modifications to `/case-studies/nordstorm/index.html` and `content/zh/case-studies/buffer/index.html`

(maybe that is intended?)

**Comment by ten2ton** [comment]
My description isn't clear enough, and there are problems that you pointed out in the previous submission, so I'd like to fix them all.


 



------------------ 原始邮件 ------------------
发件人: "Tim Bannister"<notifications@github.com>; 
发送时间: 2019年8月22日(星期四) 凌晨1:06
收件人: "kubernetes/website"<website@noreply.github.com>; 
抄送: "Mr.韩"<568310928@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [kubernetes/website] zh-trans: addcontent/case-studies/zalando/index.html (#15974)




The changes are different from the description as there are also modifications to /case-studies/nordstorm/index.html and content/zh/case-studies/buffer/index.html
 
(maybe that is intended?)

**Comment by tengqm** [comment]
Can we split this into three PRs? The overall quality of translation is good although there are still some places where discussions still needed. Splitting this to three would help accelerate things.

File: content/zh/case-studies/buffer/index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  <!-- <h2>Solution</h2>
+      Embracing containerization, Buffer moved its infrastructure from Amazon Web Services’ Elastic Beanstalk to Docker on AWS, orchestrated with&nbsp;Kubernetes. -->
+<h2>解决方案</h2>
+拥抱容器化，Buffer 将其基础设施从亚马逊网络服务上的云应用部署转移到由 Kubernetes 进行编排的 Docker中。
+      <br>
+      <br>
+<!-- <h2>Impact</h2>
+      The new system "leveled up our ability with deployment and rolling out new changes," says Farrelly. "Building something on your computer and knowing that it’s going to work has shortened things up a lot. Our feedback cycles are a lot faster now&nbsp;too." -->
+<h2>影响</h2>
+Farrelly 说,新系统“通过推出新变化并使用容器进行快速部署大提升了我们的能力”。“在计算机上构建一些东西,并且知道它是有效的,这已经让事情简单了很多。而且我们的反馈周期现在也快了很多。”
```
[the remark this answers is not in the document]
Comment (ten2ton) [reply content/zh/case-studies/buffer/index.html]: Thanks for your help,next time I'll pay attention to improvements.

**Pull request merged**